backend/repositories/embeding_model.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy.orm import Session
+from typing import List, Tuple
+
+from models.embedding_model import EmbeddingModel
+from models.error import APIError
+
+class EmbeddingModelRepository:
+    def __init__(self, db_session: Session):
+        self.db_session = db_session
+    
+    def get_embedding_models(self, limit: int, offset: int) -> Tuple[List[EmbeddingModel], APIError | None]:
+        """
+        Get all embedding models
+        """
+        try:
+            embedding_models = self.db_session.query(EmbeddingModel).limit(limit).offset(offset).all()
+            if embedding_models:
+                return embedding_models, None
+            else:
+                return [], APIError(detail="Embedding models not found")
+        except Exception as e:
+            return [], APIError(detail=str(e))
```

Comment:
Remove pagination, should return embedding models

Reply:
> Remove pagination, should return embedding models

Done